simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/component/chargingstation/ChargeProtocolVariableAccessor.java

```diff
@@ -53,7 +53,7 @@ public String getVariableName() {
     }
 
     private SetVariableResult rejectVariable(AttributePath attributePath, CiString.CiString1000 attributeValue) {
-        return READ_ONLY_VALIDATOR.validate(attributePath, attributeValue);
+        return RESULT_CREATOR.createResult(attributePath, attributeValue, SetVariableResult.AttributeStatus.REJECTED);
```

Comment:
Make sure this PR is merged last.

Reply:
all PRs should be merged in order, otherwise there will be conflicts. this one is 5th